inst/extdata/qualification-axis-settings.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
@Yuri05 
Let me know if some of the default properties need to be changed

Reply:
looks good for me so far